README.md

```diff
@@ -103,7 +103,7 @@ used, and include a complete source example leading to the bug.
 
 Guidelines for contributing to Coq in various ways are listed in the [contributor's guide](CONTRIBUTING.md).
 
-Information about the next release is at https://github.com/coq/coq/wiki/Release-Plan
+Information about the next release (Coq 8.12) is at https://github.com/coq/coq/wiki/Release-Plan-v8.12
```

Comment:
This link is also present in the contributing guide. Maybe we should remove it to avoid having to update it twice? Or we should keep a single wiki page and just add sections for each release, which would avoid having to update the README...

Reply:
Yes, that seems like a good strategy, will amend.

Reply:
Done.